testproject/ProjectSettings/ProjectVersion.txt

```diff
@@ -1,2 +1,2 @@
-m_EditorVersion: 2019.4.17f1
-m_EditorVersionWithRevision: 2019.4.17f1 (667c8606c536)
+m_EditorVersion: 2019.4.20f1
+m_EditorVersionWithRevision: 2019.4.20f1 (6dd1c08eedfa)
```

Comment:
unnecessary changes

Reply:
While I do agree this should have never been checked in, shouldn't we make sure that this defaults to the actual LTS version and not a version that doesn't does have LTS?  Just curious why we want the previous value?

Reply:
`2019.4.17f1` was also an LTS — LTS does take minor patches/updates still.

Reply:
Personally I think this should always be updated to the latest LTS version of the min editor version so in this case 2019.4